pages/members/[slug].js

```diff
@@ -93,23 +100,33 @@ const MemberPage = ({ member, loadError }) => {
             </div>
           </div>
         }
-        <div className="columns vertical-center">
-          <div className="col">
-            <div
-              className={`profile-photo xl ${image?'has-image':'placeholder'}`}
-              style={ { backgroundImage: image && `url("${cdn}${image}-max-lg.jpg")` } }
-            >
+
+        <div className='flex flex-col md:flex-row items-start'>
+
+        <div className='flex flex-col items-start space-y-5 md:w-full'>
+
+            <Link  href={'/members'}>
+              <p className="text-lg cursor-pointer">
+            {'< All Profiles'}
+              </p>
+            </Link>
+            <div>
+              <img src={member.photo? `${cdn}${member.photo}-profile-sm.jpg` : 'Add a photo'} loading="lazy" alt="this is an image" className="w-24 mt-4 rounded-full cursor-pointer transition duration-150 transform hover:scale-110" />
```

Comment:
This is still broken. 'Add a photo' is not a valid src attribute for `<img/>`

Reply:
Use the old image path which had a higher resolution: `${cdn}${image}-max-lg.jpg`